app/assets/javascripts/controllers/container_deployment/deploy_provider/deploy-provider-review-controller.js

```diff
@@ -0,0 +1,89 @@
+  ['$rootScope', '$scope',
+  function($rootScope, $scope) {
+    'use strict';
+
+    // Find the data!
+    var next = $scope;
+    while (angular.isUndefined($scope.data)) {
+      next = next.$parent;
+      if (angular.isUndefined(next)) {
+        $scope.data = {};
+      } else {
+        $scope.data = next.wizardData;
+      }
+    }
+
+    $scope.getProviderType = function() {
+      if ($scope.data.providerType == 'openshiftOrigin') {
+        return "OpenShift Origin";
+      } else if ($scope.data.providerType == 'openshiftEnterprise') {
+        return "OpenShift Enterprise";
+      }
+    };
+
+    $scope.getProviderDescription = function() {
+      if ($scope.data.provisionOn == 'existingVms') {
+        var provider = $scope.data.providers.find(function(provider) {
+          return provider.id == $scope.data.existingProviderId;
+        });
+        return 'Use existing VMs from an existing provider: ' + (provider ? provider.name : "None");
+      }
+      else if ($scope.data.provisionOn == 'newVms') {
+        var provider = $scope.data.providers.find(function(provider) {
+          return provider.id == $scope.data.newVmProviderId;
+        });
+        return 'Use existing VMs from an existing provider: ' + (provider ? provider.name : "None");
+      }
+      else if ($scope.data.provisionOn == 'noProvider') {
+        return 'Specify a list of machines to deploy on (No existing provider';
+      }
+    };
+
+    $scope.getMasterCreationTemplate = function () {
+      var selectedTemplate = $scope.data.nodeCreationTemplates.find(function(nextTemplate) {
+        return nextTemplate.id == $scope.data.masterCreationTemplateId;
+      });
+      return selectedTemplate ? selectedTemplate.name : "None";
+    };
+
+    $scope.getNodeCreationTemplate = function () {
+      var selectedTemplate = $scope.data.nodeCreationTemplates.find(function(nextTemplate) {
+        return nextTemplate.id === $scope.data.masterCreationTemplateId;
+      });
+      return selectedTemplate ? selectedTemplate.name : "None";
+    };
+
+    $scope.getAuthenticationType = function () {
+      if ($scope.data.authentication.mode == 'all') {
+        return "Allow All";
+      } else if ($scope.data.authentication.mode == 'htPassword') {
+        return "HTPassword";
+      } else if ($scope.data.authentication.mode == 'ldap') {
+        return "LDAP";
+      } else if ($scope.data.authentication.mode == 'requestHeader') {
+        return "Request Header";
+      } else if ($scope.data.authentication.mode == 'openId') {
+        return "OpenID Connect";
+      } else if ($scope.data.authentication.mode == 'google') {
+        return "Google";
+      } else if ($scope.data.authentication.mode == 'github') {
+        return "GitHub";
+      } else {
+        return "None";
+      }
+    };
+
+    $scope.getConfigServerType = function() {
+      if ($scope.data.serverConfigType == 'standardNFS') {
```

Comment:
I added translation only for the `"None"` string, since the others are authentication terms